### DISCUSSION TITLE
Supporting Apollo Server v3?

### DISCUSSION THREAD

**Discussion by RyannGalea** [the-guild-org/apollo-angular#1716] [General]
Is this in the plans?

I'm currently updating server across and having some issues with Websockets and subscriptions now my server is on v3

**Comment by KeithGillette** [comment]
Yes, I think think the Apollo Angular Subscriptions documentation should be updated to use the Apollo-recommended `graphql-ws` WebSockets implementation. We've replaced `subscriptions-transport-ws` with `graphql-ws` in our Apollo Angular project following the [`graphql-ws` Apollo Client Web Recipe](https://github.com/enisdenjo/graphql-ws#recipes) but are still figuring out how to cleanly unsubscribe from all GraphQL Subscriptions on user sign-out, as `graphql-ws` doesn't seem to have an equivalent to the `subscriptions-transport-ws` `SubscriptionClient.unsubscribeAll()`.
